internal/policy/source/source.go

```diff
@@ -67,7 +68,10 @@ type PolicyUrl struct {
 
 // GetPolicies clones the repository for a given PolicyUrl
 func (p *PolicyUrl) GetPolicy(ctx context.Context, workDir string, showMsg bool) (string, error) {
+	// Trim scheme from repo url as this breaks go-getter.
 	sourceUrl := p.PolicyUrl()
+	sourceUrl = strings.TrimPrefix(sourceUrl, "https://")
+	sourceUrl = strings.TrimPrefix(sourceUrl, "http://")
```

Comment:
Not allowed, see https://github.com/enterprise-contract/ec-cli/blob/2bbed8f8295f2b87fc5c64fb7e7c0bb774a25443/internal/downloader/downloader.go#L49C1-L51.

Reply:
Ack.